libs/blocks/mobile-app-banner/mobile-app-banner.js

```diff
@@ -48,8 +59,18 @@ function branchInit(key) {
       0
     );
     const privacyConsent = window.adobePrivacy?.hasUserProvidedConsent();
+    const isAndroid = navigator.userAgent.includes('Android');
+    
+    const cookieGrp = window.adobePrivacy?.activeCookieGroups();
+    const performanceCookieConsent = cookieGrp.includes('C0002');
+    const advertisingCookieConsent = cookieGrp.includes('C0004');
+
+    if ( performanceCookieConsent && advertisingCookieConsent && isAndroid ) {
```

Comment:
Does the linter not complain? 🤔 We don't leave spaces after `if` braces usually
```suggestion
    if (performanceCookieConsent && advertisingCookieConsent && isAndroid) {
```

Reply:
For some reason, linter wasn't working for me.. checked on that now. I have made the suggested changes. 
Thanks a lot!